src/componnets/CardCart.js

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import classes from "./Card.module.css";
+
+const CardCart = ({ productData, onDelete, onEditProductAmount }) => {
```

Comment:
I would expect this to be named the other way round: "`CartCard`". Funny name anyway, but sometimes these kind of names are hard to avoid.